taxi/forms.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+def get_license_validator() -> object:
```

Comment:
would be better to create function `clean_license_number` in `DriverLicenseUpdateForm` for validation